Jenkinsfile

```diff
@@ -12,11 +12,6 @@ pipeline {
                     // Clean up any unwanted files lying about after the previous build.
```

Comment:
The https://github.com/GSS-Cogs/csvcubed-devtools project do not have the tests folder. We should make sure the tests folder is also added to the project structure described in https://app.zenhub.com/workspaces/features-squad-61b72275ac896c0010a1b00b/issues/gss-cogs/csvcubed/454